### ISSUE TITLE
Condition off column top margin

### ISSUE THREAD

**Issue by jimmynotjim** [cfpb/cf-layout#9]
Currently sibling columns have a default top margin that is removed with a min-width media query that has to specify every possible column combination https://github.com/cfpb/cf-layout/blob/gh-pages/src/cf-layout.less#L453-L466

To reduce complexity, a max-width media query should be set to add the margin to the column siblings.

ex

``` css
.content-l_col {
    .respond-to-max(@tablet-min, {
        & + & {
            margin-top: unit(@grid_gutter-width / @base-font-size-px, em);
        }
    });
}
```

My only concern is how this might affect the `.stack-col()` mixin https://github.com/cfpb/cf-layout/blob/gh-pages/src/cf-layout.less#L578-L617. Does anyone have examples of that in use I could test on?

**Comment by himedlooff** [comment]
`.stack-col()` is used in `.content-l__main`. In cfgov-refresh there are a bunch of things using it that you could test on. Does that help?

**Comment by jimmynotjim** [comment]
This was addressed in #15 [L:529](https://github.com/cfpb/cf-layout/pull/15/files#diff-9f1ee77d4c339e07b8ea9e8db1b74193R529)
